package.json

```diff
@@ -8,9 +8,9 @@
   },
   "dependencies": {
     "@babel/polyfill": "7.4.4",
```

Comment:
This commit should be squashed with the first in the series which introduces these new modules.

Reply:
Fixed